website/docs/r/vapp_network.html.markdown

```diff
@@ -50,7 +50,11 @@ The following arguments are supported:
 * `name` - (Required) A unique name for the network.
 * `description` - (Optional; *v2.7+*, *vCD 9.5+*) Description of vApp network
 * `vapp_name` - (Required) The vApp this network belongs to.
-* `netmask` - (Optional) The netmask for the new network. Default is `255.255.255.0`.
+* `netmask` - (Deprecated) Use `prefix_length` instead. The netmask for the new network.
+
```

Comment:
I can't comment in exact place, but as we are deprecating netmask, you should update the "## Example usage" snippet to use prefix_length. We avoid using deprecated attributes in our sample snippets.

Also, IMO, you could add another example snippet with IPv6 usage

Reply:
Fixed.